src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -70,9 +80,15 @@ export interface MenuItemProps extends VibeComponentProps {
    *    2. click/hover on icon button will open the sub menu
    */
   splitMenuItem?: boolean;
+  "aria-label"?: AriaAttributes["aria-label"];
 }
 
-const MenuItem: VibeComponent<MenuItemProps> & {
+interface MenuItemTitleComponentProps extends Omit<MenuItemProps, "title"> {
```

Comment:
@YossiSaadi changed it eventually because MenuItemProps is exported so it breaks stuff